tests/test_livysession.py

```diff
@@ -436,6 +436,22 @@ def test_create_sql_hive_context_happens_once(self):
                in http_client.post_statement.call_args_list
         assert len(http_client.post_statement.call_args_list) == 1
 
+        session._get_sql_context_creation_command = MagicMock()
+
+        session._get_sql_context_creation_command.return_value.execute.return_value = (True, "Success")
+        session.created_sql_context = None
+        session.create_sql_context()
+        assert session.created_sql_context
+
+        session._get_sql_context_creation_command.return_value.execute.return_value = (False, "Exception")
+        session.created_sql_context = None
+
+        try:
+            session.create_sql_context()
+            assert False
+        except ValueError:
+            assert session.created_sql_context is None
+
```

Comment:
message attribute is not existing in Python 3 and I believe the current test is sufficient.